### DISCUSSION TITLE
UI界面都回来力，作者大大好

### DISCUSSION THREAD

**Discussion by Fengdan-NAvia** [share121/inter-knot#4932] [灌水]
![5167b86c1ac348bde15591e080afadd](https://github.com/user-attachments/assets/68ab8b9d-fe04-468b-b08f-f90df7ced8f3)

**Comment by konghuan7759** [comment]
👍🏻

**Comment by Akanyi** [comment]
<h1>这么大的信息瀑布你看得过来吗？

**Reply by Akanyi** [reply]
个人觉得可以在电脑端信息瀑布中更改一些权重，让它不要显示过多的内容。

**Reply by Fengdan-NAvia** [reply]
是不是和我电脑屏幕有关？我电脑32寸的屏幕，其实看着还好。

**Comment by Junjun280** [comment]
是挺不戳的
